.github/workflows/ci.yml

```diff
@@ -18,7 +18,28 @@ on:
 
 jobs:
 
-  build:
+  Format:
```

Comment:
Please use pre-commit instead of calling black manually, as done here: https://github.com/olofk/fusesoc/blob/master/.github/workflows/lint.yml

Reply:
Done.